cunumeric/array.py

```diff
@@ -29,19 +29,14 @@
 from .utils import dot_modes
 
 
-def add_boilerplate(*array_params: str, mutates_self: bool = False):
+def add_boilerplate(*array_params: str):
```

Comment:
now this function has become much simpler with this change, do you think it would be possible to merge this with the other `add_boilerplate` in `module.py`?

Reply:
[a0ff779](https://github.com/nv-legate/cunumeric/pull/357/commits/a0ff77904693d159b4736342d1b2854b63ffabf5)